### ISSUE TITLE
FlowContainer returns wrong line count

### ISSUE THREAD

**Issue by boruok** [godotengine/godot#68991]
### Godot version

v3.6.beta.custom_build [0803b4168]

### System information

PopOS 22.04 LTS, GTX1060

### Issue description

Look like `FlowContainer` `get_line_count()` always returns 0, even if they have child-nodes on em, which is incorrect behiavour.

### Steps to reproduce

1. make new scene with `HFlowContainer` or `VFlowContainer` it doesn't matter
2. attach following script

```gdscript
extends FlowContainer


func _ready() -> void:
#	yield(get_tree(), "idle_frame")
	print(get_line_count())
```
3. run scene and see output
4. uncomment `yield` line
5. see output

### Minimal reproduction project

issue easily reproducible with any project

**Comment by dzil123** [comment]
I can't reproduce this.

With this node setup:
![image](https://user-images.githubusercontent.com/5725958/203247773-e2b93b45-fb27-4415-b75b-8eaefffff44f.png)

And this script:
```gdscript
extends FlowContainer

func _ready() -> void:
	print(get_line_count())
	yield(get_tree(), "idle_frame")
	print(get_line_count())
```

The first print says `0`, and the second print says `4`. This is expected behavior, the line count is cached so changes made on this frame may not be reflected until next frame.

If this doesn't help, please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot.

**Comment by boruok** [comment]
@dzil123 i suggest to change this behiavour to make it like match other nodes (`Label`, `RichTextLabel`, `TextEdit`) they do have `get_line_count` method and they return value without waiting next frame. At least this must be documented.

btw `rect_size` also `yields`

**Comment by Zireael07** [comment]
Some other things also don't work in the first frame (e.g. some of RichTextLabel's functions). This is not an exception

**Comment by akien-mga** [comment]
Yes this behavior seems pretty standard for UI nodes, I don't think it will change. CC @godotengine/gui-nodes 

On the other hand, is "line count" a standard term for wrapped elements in a flow container? Seems a bit weird to me in a non text-related UI context.

**Comment by Geometror** [comment]
Calculating the line count immediately would almost certainly result in a wrong value due to how the layouting system works. (The size both of the FlowContainer and its children most likely changes later)

Regarding the name of the method: If I remember correctly, this has been a topic of discussion in the past. The problem is that while a name like get_row_count may be clearer for a horizontal FlowCountainer, for a vertical one it might be confusing. (But I have no strong opinion about that, so I am open for a potential rename)
